.stitch/designs/dashboard_desktop.html

```
@@ -0,0 +1,327 @@
+<span class="text-2xl font-black tracking-tighter text-indigo-600 dark:text-indigo-400">SkillSwap</span>
+<nav class="hidden md:flex items-center gap-8 font-['Plus_Jakarta_Sans'] font-medium text-sm tracking-tight">
+<a class="text-indigo-600 dark:text-indigo-400 font-bold border-b-2 border-indigo-600 pb-1" href="#">Dashboard</a>
+<a class="text-[#5e5680] dark:text-slate-400 hover:text-indigo-500 transition-colors" href="#">Matches</a>
+<a class="text-[#5e5680] dark:text-slate-400 hover:text-indigo-500 transition-colors" href="#">Chat</a>
+<a class="text-[#5e5680] dark:text-slate-400 hover:text-indigo-500 transition-colors" href="#">Profile</a>
+</nav>
+</div>
+<div class="flex items-center gap-4">
+<div class="hidden lg:flex items-center bg-surface-container-high rounded-full px-4 py-2 gap-2">
+<span class="material-symbols-outlined text-on-surface-variant text-lg">search</span>
+<input class="bg-transparent border-none focus:ring-0 text-sm w-64 placeholder:text-on-surface-variant" placeholder="Search skills or mentors..." type="text"/>
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)